o2a/utils/relation_utils.py

```diff
@@ -26,7 +26,7 @@ def chain(ops: Sequence[Task]) -> List[Relation]:
 
     ** Example: **
 
-    :codeblock: pycon
+    :codeblock: python
```

Comment:
Why did you change it?
<img width="797" alt="Screenshot 2019-05-23 at 10 17 52" src="https://user-images.githubusercontent.com/12058428/58236878-28f67780-7d44-11e9-8504-511cd3e8f132.png">

Reply:
oups :).

Reply:
My bad :). Did it while fixing spelling .